resolwe_bio/processes/import_data/nucleotide_seq.yml

```diff
@@ -7,6 +7,9 @@
   name: Upload nucleotide sequence(s) (FASTA)
   requirements:
     expression-engine: jinja
+    executor:
+      docker:
+        image: "resolwe/bio-linux8-resolwe-preview:icount"
```

Comment:
For any test that relies on this kind of object as an input, we will have the same problem as with the icount tests. Both the default and the icount docker images will get downloaded. Based on the possible @tjanez recommendation, check if any other test will have to be temporary disabled.